scenarios/04_transitive_via_export/BUILD

```diff
@@ -0,0 +1,11 @@
+package(default_visibility = ["//03_basic:__subpackages__"])
+
+load("@rules_intellij_generate//:def.bzl", "intellij_modules_xml")
```

Comment:
what does def.bzl mean?

Reply:
after that thread I started on bazel-discuss, I looked around at some of the goog projects and decided to copy the style of rules_go